core/variant/variant.h

```diff
@@ -932,14 +932,9 @@ const Variant::ObjData &Variant::_get_obj() const {
 template <typename... VarArgs>
 String vformat(const String &p_text, const VarArgs... p_args) {
 	Variant args[sizeof...(p_args) + 1] = { p_args..., Variant() }; // +1 makes sure zero sized arrays are also supported.
```

Comment:
Is there no way to not even construct this ? I don't know if C++ forces variadic arguments to be declared in a contiguous fashion ?

Reply:
No idea! I haven't found anything, but it would be cool to know!